examples/rgbpp/.env.example

```
@@ -0,0 +1,28 @@
+# Ture for CKB and BTC Mainnet and false for CKB and BTC Testnet, the default value is false
+IS_MAINNET=false
+
+# CKB Variables
+
+# The CKB secp256k1 private key whose format is 32bytes hex string with 0x prefix
+CKB_SECP256K1_PRIVATE_KEY=0x-private-key
+
+# CKB node url which should be matched with IS_MAINNET
+CKB_NODE_URL=https://testnet.ckb.dev/rpc
+
+# CKB indexer url which should be matched with IS_MAINNET
+CKB_INDEXER_URL=https://testnet.ckb.dev/indexer
+
+
+# BTC Variables
+
+# The BTC private key whose format is 32bytes hex string without 0x prefix
+BTC_PRIVATE_KEY=private-key
+
+# The BTC assets api url which should be matched with IS_MAINNET
+VITE_BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro;
```

Is the ";" at the end valid? Same question for below variables.
```suggestion
VITE_BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro
```

---

removed